models.py

```diff
@@ -28,6 +28,30 @@ def forward(self, sequence):
     def get_name(self):
         return "BaseLine_{}".format(self.hidden_size)
 
+class PureGRU(nn.Module):
+
+    def __init__(self, **kwargs):
+        """
+        Initialize new PureGRU model.
+        :keyword arguments:
+        hidden_size: int, number of hitten units.
+        num_layers: int, Number of recurrent layers
+        """
+        super().__init__()
+        self.hidden_size = kwargs["hidden_size"]
+        self.num_layers = kwargs["num_layers"]
+
+        self.gru = nn.GRU(input_size=256, hidden_size=self.hidden_size, num_layers=self.num_layers)
+        self.output_layer = nn.Linear(self.hidden_size, 4)
+        self.h0 = nn.Parameter(torch.randn(self.num_layers, 1, self.hidden_size))
+
+    def forward(self, sequence):
+        output, hn = self.gru(sequence, self.h0)
+        predictions = self.output_layer(output)
+        return predictions
+
+    def get_name(self):
+        return "PureGRU_h:{}_l:{}".format(self.hidden_size, self.num_layers)
```

Comment:
Do we really want to have ':' in the model names?

Reply:
Hmm, should be ok and makes it quite readable/easy to parse automatically but something like - might be better?